app/kumactl/cmd/install/install_transparent_proxy.go

```diff
@@ -259,56 +247,56 @@ func findUidGid(uid, user string) (string, string, error) {
 	return u.Uid, u.Gid, nil
 }
 
-func parseArgs(cmd *cobra.Command, args *transparentProxyArgs) (config.Config, error) {
+func parseArgs(cfg *config.Config, args *transparentProxyArgs) error {
```

Comment:
IMO this could just return a new config

Reply:
To be honest I would like to get rid of `transparentProxyArgs` completely soon, that's why I didn't spent a lot of time on improving it.